modules/workstation/homed.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, config, ... }: {
+  config = lib.mkIf config.modules.workstation.enable {
+    services.homed.enable = true;
+  };
+
+  age = {
+    secrets = {
+      homed_497a = {
+        file = ../../secrets + "/homed_497a.age";
+        path = "/var/lib/systemd/home/497a.public";
+      };
+    };
+  };
```

Comment:
Please reformat this to `age.secrets.home_497a =` because that is more in line with the styling of the rest of this config